src/controllers/meteors.ts

```diff
@@ -0,0 +1,65 @@
+import * as service from '../services/neoWs';
+import { Request, Response, NextFunction } from 'express';
+import { Nasa, Home } from '../network/models/nasa/meteor';
+import { encode } from 'node:querystring';
+import path from 'path';
+
+export const getStartPage = async (req: Request, res: Response) => {
+  res.render(path.join(__dirname, '..', 'views', 'meteors/meteors-search.html'), {
+    pageTitle: 'Meteors Search Page',
+    path: '/',
+  });
+};
+
+type QueryRest = { [key: string]: string | undefined };
+
+export const getMeteors = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { is_potentially_hazardous_asteroid = '', is_counted = '', ...rest } = req.query;
```

Comment:
Could you consider the approach, when we save validated and parsed param in req.meta.params object?

```
const validateRequestSchema =  (schema: AnySchema): RequestHandler => {
  return async (req: Request, res: Response, next: NextFunction) => {
    try {
      removeEmptyQueryParams(req.query);
      const dataType = Object.keys(req.body).length === 0 ? 'query' : 'body';
      (req as any).meta.queryParamObject = await schema.validateAsync(req[dataType]);
      next();
    } catch (e) {
      next(e)
    }
  };
};
```

after that there you need just read this object